tests/integration/proposals/getProposals.optimism.test.ts

```diff
@@ -0,0 +1,25 @@
+import { fetchProposals } from "@/app/api/common/proposals/getProposals";
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+describe("getProposals integration tests for Optimism", () => {
+  beforeAll(async () => {
+    // Set the environment variable for the Optimism tenant
+    process.env.NEXT_PUBLIC_AGORA_INSTANCE_NAME = "scroll";
```

Comment:
We can update Tenant class to accept `namespace` to override `NEXT_PUBLIC_AGORA_INSTANCE_NAME`: https://github.com/voteagora/agora-next/blob/main/src/lib/tenant/tenant.ts#L25